### PR TITLE
remarks pills

### PR DESCRIPTION
**Pull request by SidelineCory24**
image

### CODE REVIEW

**Comment by mjoyce91** [comment]
This is what happens when there are more remarks and the screen width is decreased

Screen Shot 2021-12-13 at 12 51 08 PM

File: src/Components/Agenda/RemarksPill/RemarksPills.test.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { shallow } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+import RemarksPill from './RemarksPill';
+
+describe('RemarksPill', () => {
```
Comment (mjoyce91) [inline src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:5]: Can we add a test in here that checks the text? For example, performing a check that the text and backgroundColor match the provided prop's values

File: src/Components/Agenda/RemarksPill/RemarksPill.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  );
+};
+
+RemarksPill.propTypes = {
+  fakeRemark: PropTypes.arrayOf(PropTypes.shape({})),
+};
+
+
+RemarksPill.defaultProps = {
+  fakeRemark: [],
```
Comment (mjoyce91) [inline src/Components/Agenda/RemarksPill/RemarksPill.jsx:18]: Let's just name this prop to `remark`

File: src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx [hunk 3]
```diff
@@ -126,6 +150,17 @@ const AgendaItemLegs = props => {
           }
         </tbody>
       </table>
+      {
+        !isCard &&
+        <div className="remarks-container">
+          <div className="remarks-text">Remarks:</div>
+          {
+            fakeRemarks.map(fakeRemark => (
+              <RemarksPill fakeRemark={fakeRemark} />
```
Comment (mjoyce91) [inline src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx:159]: Let's add a React `key`

File: src/Components/Agenda/RemarksPill/RemarksPill.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/Agenda/RemarksPill/RemarksPill.jsx]: done

File: src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx (hunk 3 above)
Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx]: done

File: src/Components/Agenda/RemarksPill/RemarksPills.test.jsx (hunk 1 above)
Comment (SidelineCory24) [reply src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:5]: updated the test

**Comment by mjoyce91** [comment]
Does the "Remarks:" text seem a little big compared to the rest of the card's text? I made the change below and I think it looks a little better:

Before:
Screen Shot 2021-12-14 at 1 11 59 PM

After (changed font-size and added margin-top):
Screen Shot 2021-12-14 at 1 13 53 PM
Screen Shot 2021-12-14 at 1 13 37 PM

File: src/Components/Agenda/RemarksPill/RemarksPills.test.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  it('matches snapshot', () => {
+    const wrapper = shallow(<RemarksPill />);
+    expect(toJSON(wrapper)).toMatchSnapshot();
+  });
+
+  it('matches snapshot when remarks are provided', () => {
+    const wrapper = shallow(<RemarksPill {...props} />);
+    expect(toJSON(wrapper)).toMatchSnapshot();
+  });
```
Comment (mjoyce91) [inline src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:25]: This is good to have, but not quite what I was asking for. Here's an example from `DarkModeToggle.test.jsx`:

```jsx
it('it displays "Disable" when isDarkMode === true', () => {
    const wrapper = shallow(
      <DarkModeToggle.WrappedComponent {...props} isDarkMode />,
    );
    expect(wrapper.find('button').text()).toBe('Disable Dark Mode');
  });

  it('it displays "Enable" when isDarkMode !== true', () => {
    const wrapper = shallow(
      <DarkModeToggle.WrappedComponent {...props} />,
    );
    expect(wrapper.find('button').text()).toBe('Enable Dark Mode');
  });
```

Comment (SidelineCory24) [reply src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:25]:
```
  it('displays a remark when remark.remark !== null', () => {
    const wrapper = shallow(<RemarksPill {...props} />);
    expect(wrapper.find('div').text()).toBe('Opts for SND');
  });
```

Comment (SidelineCory24) [reply src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:25]: @mjoyce91 is this the route you were thinking?

Comment (mjoyce91) [reply src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:25]: @SidelineCory24 yes! 👍

File: src/sass/_agenda.scss [hunk 5]
```diff
@@ -488,6 +488,29 @@
...
+          .remarks-text {
+            float: left;
+            font-size: 1.1em;
+            margin-right: .7rem;
+            margin-top: 3px;
+          }
+
+          .remarks-pill-container {
+            border-radius: 12px;
+            color: $color-white;
```
Comment (elizabeth-jimenez) [inline src/sass/_agenda.scss:506]: I would rename this - `remarks-pill-container` is for a single pill - we tend to name containers for classes that contain similar items. Maybe `remarks-pill`?

Comment (SidelineCory24) [reply src/sass/_agenda.scss]: done

File: src/Components/Agenda/RemarksPill/RemarksPills.test.jsx (hunk 4 above)
Comment (SidelineCory24) [reply src/Components/Agenda/RemarksPill/RemarksPills.test.jsx:25]: @elizabeth-jimenez  @mjoyce91 fix has been pushed, thanks again! 🐐🐐

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**